File: questão1.c
#include <stdio.h>
int main(){
    char nome[60];
    float si, sf, a;
    printf("Nome: ");
    fflush(stdin);
    fgets(nome, 60, stdin);
    printf("Salário atual: ");
    scanf("%f", &si);
    if(si<=400){a=1.15;}
    else if(si<=700){a=1.12;}
    else if(si<=1000){a=1.10;}
    else if(si<=1800){a=1.07;}
    else if(si<=2500){a=1.04;}
    else{a=1.0;}
    sf=si*a;
    a=100*(a-1.0);
    printf("\nNome: %sPorcentagem de aumento: %.2f\nSalário atual: %.2f\nNovo salário: %.2f\n", nome, a, si, sf);
    return 0;
}

File: questão1.py
nome=str(input("Nome do funcionário: "))
si=float(input("Salário atual: "))
if(si<0):
    print("Salário inválido: valor negativo.")
else:
    if(si<=400):
        a=1.15
    elif(si<=700):
        a=1.12
    elif(si<=1000):
        a=1.10
    elif(si<=1800):
        a=1.07
    elif(si<=2500):
        a=1.04
    else:
        a=1.00
    sf=si*a
    print('\nNome: %s\nPorcentagem de aumento: %.2f\nSalário atual: %f\nNovo salário: %f\n' %(nome, 100*(a-1.0), si, sf))